### ISSUE TITLE
Zuul tests always fail

### ISSUE THREAD

**Issue by felixfontein** [ansible-collections/community.general#402]
##### SUMMARY
They don't install the required collections.

Please ignore these errors in CI until they are fixed!

CC @gundalow @Andersson007 @Akasurde @abadger 

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
CI

##### ANSIBLE VERSION
```paste below
devel
```

**Comment by ansibullbot** [comment]
Files identified in the description:
None

If these files are inaccurate, please update the `component name` section of the description or use the `!component` bot command.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by goneri** [comment]
I see this error in the job output:

```
b'ERROR! module community.general.kubevirt_vm missing  documentation (or  could not  parse documentation): unknown doc_fragment(s) in file /tmp/tmpaspeeglr/ansible_collections/community/general/plugins/modules/kubevirt_vm.py: community.kubernetes.k8s_auth_options\n
```

**Comment by felixfontein** [comment]
@goneri that's because the zuul tests do not install required collection dependencies.

**Comment by goneri** [comment]
@felixfontein where is this dependency documented?

**Comment by felixfontein** [comment]
@goneri here: https://github.com/ansible-collections/community.general/blob/master/galaxy.yml#L14

**Comment by goneri** [comment]
Oh, my bad. Thanks.

**Comment by pabelanger** [comment]
Late to the game, issue here, is going to quickly run into cross-project dependencies on documentation. Which means, for even to publish a collection, you now need to make sure documentation in k8s collection is valid.

IMO, the better way to do this, is disable the ability for galaxy-importer to check for docs outside of a collection.  Enforcing a hard dependency between 2 projects, that are not managed by same team is complicated.

**Comment by pabelanger** [comment]
okay, I've reverted the galaxy-importer change, build-ansible-collections now works. However, imports into galaxy will still fail.  Today I'll create ansible-galaxy-importer job, which will install the required dependencies for each collection now.

**Comment by felixfontein** [comment]
@pabelanger do you mean galaxy right now cannot import collections that use doc fragments (or module_utils) from other collections? That would be a pretty huge bug. Do you know if an issue exists for that?

**Comment by pabelanger** [comment]
I don't know, we'll have to tag a release again to see. Last we tried, it was blocked on galaxy-importer, why we added it to the build collection job.  Which, caused another issue.

We'll know more by tomorrow, once the new job is up and running.

**Comment by felixfontein** [comment]
It looks like this is fixed now: the tests for #382 completely passed!

**Comment by felixfontein** [comment]
@pabelanger thanks a lot for fixing this!
